serverless/src/registration.js

```diff
@@ -1,20 +1,50 @@
 const AWS = require("aws-sdk");
-
+const { httpResponse } = require("./http_response");
 exports.handler = async (event) => {
-  const dbClient = new AWS.DynamoDB.DocumentClient();
-
-  exports.handler = async (event) => {
-    let body;
-    let statusCode = 200;
-    try {
-      body = await dbClient.put(event).promise();
-    } catch (err) {
-      statusCode = err.statusCode;
-      body = err.message;
-    }
-    return {
-      statusCode,
-      body,
-    };
+  const docClient = new AWS.DynamoDB.DocumentClient();
+  let body;
+  let statusCode = 201;
+  let headers = {
+    "Content-Type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+    "Access-Control-Allow-Methods": "OPTIONS,POST,GET",
   };
+  let obj = JSON.parse(event.body);
+  const firstName = obj.firstName;
```

Comment:
All of this is duplicate work. You will already have the item in the format that you need, so you are just resetting the properties to the same value. You should be able to set Item: obj since that object (the body) is already a JavaScript object